apps/wallet-mobile/src/features/Swap/SwapNavigator.tsx

```diff
@@ -22,83 +22,38 @@ export const SwapTabNavigator = () => {
   const {atoms, color} = useTheme()
   const isKeyboardOpen = useIsKeyboardOpen()
 
-  // state data
-  const {wallet} = useSelectedWallet()
-  const {
-    aggregatorTokenId,
-    lpTokenHeldChanged,
-    frontendFeeTiers,
-    frontendFeeTiersChanged,
-    sellTokenInfoChanged,
-    primaryTokenInfoChanged,
-  } = useSwap()
-  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-  const lpTokenHeld = usePortfolioBalances({wallet}).records.get(aggregatorTokenId!)
-
-  // initialize sell with / and primary token
-  React.useEffect(() => {
-    sellTokenInfoChanged(wallet.portfolioPrimaryTokenInfo)
-    primaryTokenInfoChanged(wallet.portfolioPrimaryTokenInfo)
-  }, [primaryTokenInfoChanged, sellTokenInfoChanged, wallet.portfolioPrimaryTokenInfo])
-
-  // update the fee tiers
-  React.useEffect(() => {
-    frontendFeeTiersChanged(frontendFeeTiers)
-  }, [frontendFeeTiers, frontendFeeTiersChanged])
-
-  // update lp token balance
-  React.useEffect(() => {
-    if (aggregatorTokenId == null) return
-
-    lpTokenHeldChanged(lpTokenHeld)
-  }, [aggregatorTokenId, lpTokenHeld, lpTokenHeldChanged])
-
-  useSwapTokensOnlyVerified({suspense: false})
-
   const {visible: isSearchBarVisible} = useSearch()
 
+  const listeners = {
+    tabPress: (e: EventArg<'tabPress', true, undefined>) => {
+      // if the keyboard is open, the user needs to close first the keyboard
+      // then, press again the tab to change screen
+      // to avoid screen freezing
+      if (isKeyboardOpen) {
+        Keyboard.dismiss()
+        e.preventDefault()
+      }
+    },
+  }
+
   return (
     <KeyboardAvoidingView style={[styles.flex, styles.root]}>
       <SafeAreaView edges={['bottom', 'left', 'right']} style={styles.flex}>
         <Tab.Navigator
           screenOptions={({route}) => ({
             ...defaultMaterialTopTabNavigationOptions(atoms, color),
             ...(isSearchBarVisible && {tabBarStyle: {height: 0}}),
-            tabBarLabel: route.name === 'token-swap' ? strings.tokenSwap : strings.orderSwap,
+            tabBarLabel: {'token-swap': strings.tokenSwap, orders: strings.orderSwap, 'manager-config': 'Manager'}[
```

Comment:
- [ ] Fix: according product requirements (in-progress)

Reply:
Nav was changed already